File: src/main/java/com/trade/securities/application/port/in/GetFscStockPriceUseCase.java
package com.trade.securities.application.port.in;

import java.util.Map;

public interface GetFscStockPriceUseCase {
    Map<String, Object> getStockPrices(int pageNo, int numOfRows, String basDt, String itmsNm, String likeItmsNm);
}

File: src/main/kotlin/com/trade/securities/application/port/in/GetFscStockPriceUseCase.kt
package com.trade.securities.application.port.`in`

interface GetFscStockPriceUseCase {
    fun getStockPrices(pageNo: Int, numOfRows: Int, basDt: String, itmsNm: String?, likeItmsNm: String?): Map<String, Any?>
}
